reconcile/openshift_saas_deploy.py

```diff
@@ -67,8 +67,17 @@ def slack_notify(
     in_progress: bool,
     trigger_integration: Optional[str] = None,
     trigger_reason: Optional[str] = None,
+    skip_successful_notifications: Optional[bool] = False,
 ) -> None:
     success = not ri.has_error_registered()
+    # if the deployment doesn't want any notifications for successful
+    # deployments, then we should grant the wish. However, there's a user
+    # expereince concern where the deployment owners will receive a "in
+    # progress" notice but no subsequent notice. We handle this case by
+    # including an "fyi" message for in progress deployments down below.
+    if success and skip_successful_notifications and not in_progress:
+        logging.info(f"Skipping Slack notification for {saas_file_name} to {env_name} because deploy was successful.")
```

Comment:
I would use `logging.debug` instead. Those info logs will be shown in our #sd-app-sre-econcile slack channel.

Reply:
@chassing Ooh, thanks. I didn't realize those make it into Slack. Is `debug` log level the preferred way to put these messages into the pods' logs (or elsewhere so they are generally available if one went to look for them)?

Reply:
It maybe ok here to use info log, as this integration is run tekton, the log message won't show in slack channel.

Reply:
Ah yes, you're right. I missed that point 🙇